src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let onlyMens;
+
+  century
+    ? onlyMens = people.filter(person => person.sex === 'm'
+      && Math.ceil(person.died / 100) === century)
+    : onlyMens = people.filter(person => person.sex === 'm');
```

Comment:
Now, it's little difficult for me to understand all condition. You should create variables for these conditions.
Checklist p3.